contributing/topics/defaults-and-breaking-changes.md

```diff
@@ -0,0 +1,128 @@
+# Defaults and Breaking Changes
```

Comment:
We don't have a guide on breaking changes, could we rename this to `guide-breaking-changes` and have `Defaults` as a subsection within it?
```suggestion
# Guide: Breaking Changes
```